api/notd/api/response_builder.py

```diff
@@ -75,3 +76,15 @@ async def token_transfer_from_model(self, tokenTransfer: TokenTransfer) -> ApiTo
 
     async def token_transfers_from_models(self, tokenTransfers: Sequence[TokenTransfer]) -> Sequence[TokenTransfer]:
         return await asyncio.gather(*[self.token_transfer_from_model(tokenTransfer=tokenTransfer) for tokenTransfer in tokenTransfers])
+
+    async def retrieve_highest_priced_transfer(self, transfer: TokenTransfer) -> ApiTokenTransfer:
```

Comment:
you dont need these functions. just use `token_transfer_from_model` where needed in the api file

Reply:
(or collection_token_from_registry_address_token_id)